include/bpf/libbpf.h

```diff
@@ -917,6 +917,31 @@ void
 ring_buffer__free(struct ring_buffer* rb);
 /** @} */
 
+/**
+ * @brief Query the BPF program flags.
```

Comment:
I think this needs some clarification in the doxygen.  If I understand correctly, it does not query the extension so cannot say what the actual flags are currently there, only flags value last known to this user mode DLL instance, right?